### ISSUE TITLE
Move libOpenCLHybrid into its own repo

### ISSUE THREAD

**Issue by rcombs** [bashbaug/SimpleOpenCLSamples#18]
You'd mentioned wanting to do this in [this comment](https://github.com/KhronosGroup/OpenCL-ICD-Loader/issues/107#issuecomment-640356264) and it'd be super useful to me, mind setting that up?

**Comment by bashbaug** [comment]
Hi @rcombs, thanks for the reminder and nudge - it's great to know this is still interesting and useful.

I'm still planning to do this but I probably won't get to it for another week or so.  Would that work for you?

**Comment by rcombs** [comment]
Sure, that's fine! Thanks much.

**Comment by bashbaug** [comment]
Hi, sorry for the delay.  I haven't had as much time to work on this as I would have liked to, but I did get _something_ pushed out at least:

https://github.com/bashbaug/opencl-runtime-loader

I've verified that these simple samples build and run when they link against this library instead of the standard OpenCL ICD loader.

I have plenty of plans to enhance this library but hopefully this is good enough to get started for now.  If you find bugs or have requests for enhancements, feel free to file issues on that repo (I'll also take pull requests!).  Thanks!
